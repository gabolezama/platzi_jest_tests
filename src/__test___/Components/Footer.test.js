import React from "react";
import { mount, shallow, render } from "enzyme";
import Footer from "../../components/Footer";


describe("<Footer/>", function(){
    test('Render del Footer', () => {
        const footer = mount(<Footer/>);
        expect(footer.length).toEqual(1);
    })
})

describe("<Footer/> Snapshot", ()=>{
    test('snapshot test for Footer', () => {
        const footer = render(<Footer />);
        expect(footer).toMatchSnapshot();
      });
});