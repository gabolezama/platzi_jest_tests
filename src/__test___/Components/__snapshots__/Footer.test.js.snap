// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Snapshot snapshot test for Footer 1`] = `
LoadedCheerio {
  "0": <div
    class="Footer"
  >
    <p
      class="Footer-title"
    >
      Platzi Store
    </p>
    <p
      class="Footer-copy"
    >
      Todos los Izquierdos Reservados
    </p>
  </div>,
  "_root": LoadedCheerio {
    "0": Document {
      "children": [
        <html>
          <head />
          <body />
        </html>,
      ],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
      "x-mode": "quirks",
    },
    "_root": [Circular],
    "length": 1,
    "options": {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
